TrueDialog.Client.Standard.WebTests/Startup.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TrueDialog.Configuration;
+
+namespace TrueDialog.Client.Standard.WebTests
+{
+    public class MyConfigProvider : ITrueDialogConfigProvider
+    {
+        public ITrueDialogConfig GetConfig()
+        {
+            return new TrueDialogConfig
+            {
+                Username = "Integraton_User_1",
+                Password = "Pivot123!123"
+            };
+        }
+    }
+
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.UseTrueDialog<MyConfigProvider>();
```

Comment:
Most people are going to be used to how .NET Core hosting does things.  Using IOptions<TrueDialogConfig> in our classes that need it, and then here we'd have something akin to:
`services.UseTrueDialog(cfg => {
    cfg.Username("Integration_User_1");
    cfg.Password("Pivot123!123");
});`